src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -271,12 +271,25 @@ export default function ToeristischeVerhuur() {
             className={styles.SectionBorderTop}
             startCollapsed={false}
             hasItems={!!plannedVerhuur.length}
-            noItemsMessage="Er is geen geplande verhuur gevonden."
+            noItemsMessage={
+              is2021
+                ? `U ziet hieronder alleen meldingen die vanaf 8 april zijn ingepland.
```

Comment:
Volgens mij kun je dit tekstje wel in een constant zetten buiten het component. 

```
const VERHUUR_DISCLAIMER_2021 = `U ziet hieronder alleen meldingen die vanaf 8 april zijn
                ingepland. Meldingen die u vóór 8 april hebt ingepland kunnen
                niet worden getoond.`;

Reply:
Tekstjes zijn wel degelijk dusdanig anders

Reply:
Er zijn er toch 4 hetzelfde?